### PR TITLE
Release Godot 4.2 RC 1 and publish release notes

### PR DESCRIPTION
**Pull request by akien-mga**

### CODE REVIEW

File: collections/_article/release-candidate-godot-4-2-rc-1.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+## Downloads
+
+{% include articles/download_card.html version="4.2" release="rc1" article=page %}
+
+**Standard build** includes support for GDScript and GDExtension.
+
+**.NET build** (marked as `mono`) includes support for C#, as well as GDScript and GDExtension.
+- .NET build requires [.NET SDK 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) or [7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) installed in a standard location.
+  - [.NET 8.0](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) should also be supported, but keep in mind this version of the .NET SDK is still in preview. Give it a try and let us know if you find any bugs.
```
Comment (raulsntos) [inline collections/_article/release-candidate-godot-4-2-rc-1.md]: .NET 8.0 is no longer in preview :slightly_smiling_face: 

```suggestion
- .NET build requires [.NET SDK 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0), [7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0), or [8.0](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) installed in a standard location.
```

File: collections/_article/release-candidate-godot-4-2-rc-1.md [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+author: Yuri Sizov
+image: /storage/blog/covers/release-candidate-godot-4-2-rc-1.webp
+image_caption_title: "Forest River 2"
+image_caption_description: "A concept scene by Beau Seymour"
+date: 2023-11-17 15:00:00
 ---
 
+We are happy to announce that all major issues identified during the beta testing phase have been resolved, and Godot 4.2 starts to look seaworthy. This means it is time for the first Release Candidate of the cycle.
+
+As always, please, remember that this is still pre-production software. We don't expect any new blockers to appear at this stage, but we still need to verify, with your help, that Godot 4.2 is ready. You are encouraged to give it a try, and if all goes according to the plan the final release should reach the shelves of your local store before the end of the next week.
```
Comment (JohnVeness) [inline collections/_article/release-candidate-godot-4-2-rc-1.md]:
```suggestion
As always, please, remember that this is still pre-production software. We don't expect any new blockers to appear at this stage, but we still need to verify, with your help, that Godot 4.2 is ready. You are encouraged to give it a try, and if all goes according to plan the final release should reach the shelves of your local store before the end of next week.
```

File: collections/_article/release-candidate-godot-4-2-rc-1.md [hunk 3]
```diff
@@ -1,19 +1,27 @@
...
+
+A number of crashes and regressions from previous changes was fixed for RC1. Keep an eye for improvements in animation, navigation, and tilemaps. We also reverted one change introduced as early as [dev snapshot 1](/article/dev-snapshot-godot-4-2-dev-1)! The default shortcut for saving scripts and shaders is no longer overriding the shortcut for saving scenes. You can still configure whichever behavior you prefer in the editor settings.
```
Comment (JohnVeness) [inline collections/_article/release-candidate-godot-4-2-rc-1.md]:
```suggestion
A number of crashes and regressions from previous changes were fixed for RC1. Keep an eye for improvements in animation, navigation, and tilemaps. We also reverted one change introduced as early as [dev snapshot 1](/article/dev-snapshot-godot-4-2-dev-1)! The default shortcut for saving scripts and shaders is no longer overriding the shortcut for saving scenes. You can still configure whichever behavior you prefer in the editor settings.
```

**Review by akien-mga — COMMENTED** [review]
Looks great to me, thanks for finalizing this! Great intro :)

Can't approve as I'm the PR author it seems :P

**Review by YuriSizov — APPROVED** [review]
Pretending to be Akien and Akien pretending to be me.

**Review by adamscott — APPROVED** [review]

**Comment by YuriSizov** [comment]
Thanks everyone!

**Pull request merged**